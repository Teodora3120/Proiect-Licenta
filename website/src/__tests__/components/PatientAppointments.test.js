import React from 'react'
import { shallow } from 'enzyme'
import PatientAppointment from '../../views/dashboard/PatientAppointments'
it('should test PatientAppointment component', () => {
  const wrapper = shallow(<PatientAppointment/>)
  expect(wrapper).toMatchSnapshot()
})
