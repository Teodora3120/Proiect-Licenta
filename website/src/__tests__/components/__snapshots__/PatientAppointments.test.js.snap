// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test PatientAppointment component 1`] = `
<Fragment>
  <Header />
  <Container
    className="mt--7"
    fluid={true}
    tag="div"
  >
    <Row
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Col
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Card
          className="shadow"
          tag="div"
        >
          <CardHeader
            tag="div"
          >
            <Row
              className="align-items-center"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <Col
                className="text-left"
                lg="8"
                md="8"
                sm="12"
                tag="div"
                widths={
                  Array [
                    "xs",
                    "sm",
                    "md",
                    "lg",
                    "xl",
                  ]
                }
                xs="12"
              >
                <h3
                  className="mb-0"
                >
                  View your appointments
                </h3>
              </Col>
            </Row>
          </CardHeader>
          <CardBody
            tag="div"
          >
            <Row
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <Table
                className="align-items-center table-flush"
                responsive={true}
                responsiveTag="div"
                tag="table"
              >
                <thead
                  className="thead-light"
                >
                  <tr>
                    <th
                      scope="col"
                    >
                      Date
                    </th>
                    <th
                      scope="col"
                    >
                      Time
                    </th>
                    <th
                      scope="col"
                    >
                      Doctor
                    </th>
                    <th
                      scope="col"
                    >
                      Email
                    </th>
                    <th
                      scope="col"
                    >
                      Prescription
                    </th>
                  </tr>
                </thead>
                <tbody />
              </Table>
            </Row>
          </CardBody>
        </Card>
      </Col>
    </Row>
  </Container>
</Fragment>
`;
